README.md

```diff
@@ -27,8 +27,7 @@ Change Log
         * Prints the commands submitted to the cluster to `STDOUT` when launched.
     * New `--runfile_prefix` option to help avoid potential filename clashes.
     * Added compatability with GRIDEngine `~/.sge_request files` (by ignoring them). Thanks to [@s-andrews](https://github.com/s-andrews)
-    * New tophat module called `tophat` which introduces a workaround for buggy MAPQ reporting by tophat whilst keeping unique alignments. Thanks to [@FelixKrueger](https://github.com/FelixKrueger).
-        * The previous tophat module is still available if you're not interested in MAPQ scores and would like slightly faster processing. This is now called `tophat_broken_MAPQ.cfmod`.
+    * New tophat module called `tophat_fixedmapq` which introduces a workaround for buggy MAPQ reporting by tophat whilst keeping unique alignments. Thanks to [@FelixKrueger](https://github.com/FelixKrueger).
```

Comment:
I can't see this module? Your fork [seems to](https://github.com/s-andrews/clusterflow/tree/master/modules) have `tophat.cfmod` and `tophat_broken_MAPQ.cfmod`, the same as mine.

Are you suggesting that we should rename `tophat_broken_MAPQ.cfmod` back to `tophat.cfmod` and rename the new version to `tophat_fixedmapq.cfmod`?